CMakeLists.txt

```diff
@@ -585,7 +589,11 @@ if (USE_PYTHON)
             execute_process(
                 COMMAND cp -rn ${PY_PREFIX}/lib/${PYTHON_SUBDIR_MINOR} ${TARGET_PREFIX}/lib 
                 )
-            
+            # copy flow123d python scripts and modules
+            execute_process(
+                COMMAND cp -rT ${CMAKE_SOURCE_DIR}/src/python ${TARGET_PREFIX}/lib/flow123d 
```

Comment:
Add comment to installation of flow libraries to put them also into lib/flow123d subdirectory.